arangod/RocksDBEngine/RocksDBVPackIndex.h

```diff
@@ -197,32 +199,31 @@ class RocksDBVPackIndex : public RocksDBIndex {
       std::vector<int>* expanding);
 
   /// @brief helper function to insert a document into any index type
-  ErrorCode fillElement(
-      velocypack::Builder& leased, LocalDocumentId const& documentId,
-      VPackSlice doc, ::arangodb::containers::SmallVector<RocksDBKey>& elements,
-      ::arangodb::containers::SmallVector<uint64_t>& hashes);
+  ErrorCode fillElement(velocypack::Builder& leased,
+                        LocalDocumentId const& documentId, VPackSlice doc,
+                        absl::InlinedVector<RocksDBKey, 4>& elements,
+                        absl::InlinedVector<uint64_t, 4>& hashes);
```

Comment:
For a faithful port of the old code, I think this should have 8 elements. But as the other out variable has only capacity for 4 elements, it is absolutely fine :+1: